Common/Securities/Crypto/Crypto.cs

```diff
@@ -111,5 +112,25 @@ public Crypto(Symbol symbol, SecurityExchangeHours exchangeHours, Cash quoteCurr
         /// Get the current value of the security.
         /// </summary>
         public override decimal Price => Cache.GetData<TradeBar>()?.Close ?? Cache.Price;
+
+        /// <summary>
+        /// Decomposes the specified currency pair into a base and quote currency provided as out parameters
+        /// </summary>
+        /// <param name="symbol">The input symbol to be decomposed</param>
+        /// <param name="symbolProperties">The symbol properties for this security</param>
+        /// <param name="baseCurrency">The output base currency</param>
+        /// <param name="quoteCurrency">The output quote currency</param>
+        public static void DecomposeCurrencyPair(Symbol symbol, SymbolProperties symbolProperties, out string baseCurrency, out string quoteCurrency)
+        {
+            quoteCurrency = symbolProperties.QuoteCurrency;
+            if (symbol.Value.EndsWith(quoteCurrency))
+            {
+                baseCurrency = symbol.Value.RemoveFromEnd(quoteCurrency);
+            }
+            else
+            {
+                throw new InvalidOperationException($"symbol doesn't end with {quoteCurrency}");
```

Comment:
Perhaps to maintain backward compatibility for if not in SymbolDB? 
`Forex.Forex.DecomposeCurrencyPair(symbol.Value, out baseCurrency, out quoteCurrency);`